tests/python/test_pow.py

```diff
@@ -0,0 +1,22 @@
+import taichi as ti
+
+
+@ti.all_archs
+def test_pow_scan():
+  z = ti.var(ti.f32, shape=())
+  w = ti.var(ti.f32, shape=())
+  
+  @ti.kernel
+  def func(x: ti.f32, y: ti.f32):
+    z[None] = x ** y
+    w[None] = ti.pow(x, y)
+  
+  for x in [2, 1.4, 3, 5.11, 6.12, 10.0]:
+    for y in [2, 0.5, 4, 8, 1.12, -1, -9, 2.01, -3.3]:
+      print(x, '**', y)
+      func(x, y)
+      print(z[None], 'vs', x ** y)
+      assert abs(z[None] / x ** y - 1) < 0.0001
+      assert abs(w[None] / pow(x, y) - 1) < 0.0001
+
+test_pow_scan()
```

Comment:
`pytest` (invoked by `ti test_python`) will automatically collect all functions that start with `test_`, so this line is not really necessary :)